client/my-sites/site-settings/search.jsx

```diff
@@ -145,83 +147,92 @@ class Search extends Component {
 		);
 	}
 
-	render() {
-		const {
-			siteId,
-			siteIsJetpack,
-			isSearchEligible,
-			searchModuleActive,
-			fields,
-			translate,
-			customizerUrl,
-		} = this.props;
-
-		// for now, don't even show upgrade nudge
-		if ( ! fields.jetpack_search_supported && ! isSearchEligible ) {
-			const upgradeTitle = siteIsJetpack
-				? translate( 'Add faster, more advanced searching to your site with Jetpack Professional' )
-				: translate(
-						'Add faster, more advanced searching to your site with WordPress.com Business'
-				  );
-			return (
-				<div>
-					<SettingsSectionHeader title={ translate( 'Jetpack Search' ) } />
-
-					<Banner
-						description={
-							<div>
-								<p>
-									{ translate(
-										'The built-in WordPress search is great for sites without much content. But as your site grows, searches slow down and return less relevant results.'
-									) }
-								</p>
-								<p>
-									{ translate(
-										'Jetpack Search replaces the built-in search with a fast, scalable, customizable, and highly-relevant search hosted in the WordPress.com cloud. The result: Your users find the content they want, faster.'
-									) }
-								</p>
-							</div>
-						}
-						event={ 'calypso_jetpack_search_settings_upgrade_nudge' }
-						feature={ FEATURE_SEARCH }
-						plan={ siteIsJetpack ? PLAN_JETPACK_BUSINESS : PLAN_BUSINESS }
-						title={ upgradeTitle }
-					/>
-				</div>
-			);
-		}
-
+	renderUpgradeNotice() {
 		return (
-			<div className="site-settings__search-block">
-				{ siteId && <QueryJetpackConnection siteId={ siteId } /> }
-
-				<SettingsSectionHeader title={ translate( 'Jetpack Search' ) } />
+			<Fragment>
+				<Banner
+					description={
+						this.props.siteIsJetpack
+							? this.props.translate(
+									'Jetpack Search replaces the built-in search with a fast, scalable, and customizable search powered by WordPress.com. The result: Your users find the content they want, faster.'
+							  )
+							: this.props.translate(
+									'The built-in WordPress search is great for sites without much content. But as your site grows, searches slow down and return less relevant results.'
+							  )
+					}
+					event={ 'calypso_jetpack_search_settings_upgrade_nudge' }
+					feature={ FEATURE_SEARCH }
+					plan={ this.props.siteIsJetpack ? PRODUCT_JETPACK_SEARCH : PLAN_BUSINESS }
+					title={
+						this.props.siteIsJetpack
+							? this.props.translate(
+									'Add faster, more advanced searching to your site with Jetpack Search'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Add faster, more advanced searching to your site with Jetpack Professional' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/search-plan-to-perf-settings-)